sdk/messaging/azservicebus/admin/admin_client.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal"
 	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal/atom"
-	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal/utils"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal/exports"
```

Comment:
I used package named `exported` for similar changes in `azcore` as I felt it read better.  Minor issue but would be good to be consistent.  And if the majority thinks `exports` is better I can update my PR accordingly.

Reply:
Yes, I like exports. A "utils" package is explicitly frowned upon by the Go community.

Reply:
@jhendrixMSFT , went with 'exported'